Dockerfile

```diff
@@ -57,11 +57,15 @@ ARG NETWORK_ID
 
 ENV SGX_MODE=$SGX_MODE
 ENV NETWORK_ID=$NETWORK_ID
+ENV TEST_HW_WALLET=true
 
 # install python3.8, nodejs
 RUN set -e; \
     apt-get update; \
-    apt-get install -y software-properties-common; \
+    apt-get install -y software-properties-common git\
+        qemu-user-static python3-pyqt5 python3-construct python3-jsonschema python3-mnemonic python3-pyelftools \
+        gcc-arm-linux-gnueabihf libc6-dev-armhf-cross gdb-multiarch libvncserver-dev; \
+    cd /root && git clone  https://github.com/LedgerHQ/speculos.git && cd speculos && cmake -Bbuild -H. -DWITH_VNC=1 && make -C build/; \
```

Comment:
could this `git clone` be pinned against a specific Git commit / tag, so that the docker image building is somewhat reproducible (as the head commit may change)?